src/derivatives/functional-derivatives.md

```diff
@@ -0,0 +1,404 @@
+
+#### Column names
+
+Column names are unique alphanumeric values that are defined in a relevant JSON
+sidecar file. A naming convention might be to concatenate the atlas, ROI index,
+summarization method (defined below) and transformations
+(see [Time series transformations](#time-series-transformations)) using snake case.
+For example, `harvard_oxford_cortical_4_pc` could indicate ROI 4 of the Harvard-Oxford
+cortical atlas, summarized by taking the first principal component.
+
+#### Summarization methods
+
+To indicate the summarization method applied to construct a single time series
+for an ROI, the following column suffixes are defined:
+
+| Column suffix | Description of the transformation                                                                                                                            |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `_mean`       | The mean of voxel time series                                                                                                                                |
+| `_median`     | The median of voxel time series                                                                                                                              |
+| `_pc[_<i>]`   | The `i`th eigenvariate from principal component analysis, where `i` is 0 indexed. If `i` is not specified, the first component is implied (that is, `pc_0`). |
+| `_spat_reg`   | Spatial regression                                                                                                                                           |
+
+Example:
+
+```Text
+sub-001/
+    func/
+        sub-001_task-rest_run-1_timeseries.tsv
+```
+
+```Text
+white_matter_mean global_signal_mean ventricles_mean
+12                98                 11
+11                34                 53
+54                34                 34
+```
+
+```Text
+sub-001/
+    func/
+        sub-001_task-rest_run-1_timeseries.json
+```
+
+```JSON
+{
+    "SamplingFrequency": "TR",
+    "white_matter_mean": {
+        "ROI": "WhiteMatter"
+    },
+    "global_signal_mean": {
+        "ROI": "GlobalSignal"
+    },
+    "ventricles_mean": {
+        "ROI": "Ventricles"
+    }
+}
+```
+
+### Motion-related time series
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<label>/
+        func/
+            <source_entities>[_desc-<label>]_motion.tsv
+            <source_entities>[_desc-<label>]_motion.json
+```
+
+#### Column names
+
+The six basic motion parameters derived from motion correction have the
+following names and units:
+
+| Column name                     | Units   | Description            |
+| ------------------------------- | ------- | ---------------------- |
+| `trans_x`, `trans_y`, `trans_z` | mm      | Translation parameters |
+| `rot_x`, `rot_y`, `rot_z`       | radians | Rotation parameters    |
```

Comment:
Would make sense to also make recommendations for names in raw "physio motion" data for MRI

https://bids-specification.readthedocs.io/en/stable/modality-specific-files/physiological-and-other-continuous-recordings.html#recommendations-for-specific-use-cases

![image](https://github.com/user-attachments/assets/bb9c35a2-e189-423b-b2a3-6175e1b28a21)